application-templates/starter-typescript/package.json

```diff
@@ -58,7 +58,7 @@
     "@commercetools-uikit/text-field": "fec-155-react-19",
     "@commercetools-uikit/text-input": "fec-155-react-19",
     "@commercetools/sync-actions": "^5.14.0",
-    "@emotion/react": "^11.11.4",
+    "@emotion/react": "^11.14.0",
```

Comment:
@ragafus I suggest to downgrade to `11.13.5` as there are some TS errors in `11.14.0`.
We are working on a fix: https://github.com/commercetools/merchant-center-application-kit/pull/3722

Reply:
I guess this comment is not longer needed as the fix was already released and this branch refreshed with the corresponding version.

@emmenko can we close this comment?